src/internal/util/Immediate.ts

```diff
@@ -21,3 +22,9 @@ export const Immediate = {
     delete tasksByHandle[handle];
   },
 };
+
+export const ForTests = {
```

Comment:
I'd suggest renaming `ForTests` to `TestTools` and renaming `numberOfTasksToHandle` to `pending`.

Reply:
I agree